### PR TITLE
AGD Renewal

### PR DESCRIPTION
**Pull request by efecarranza**
# Changelog

Add payload to approve AAVE Grants DAO to use funds.
Add test for payload.

Proposal: https://governance.aave.com/t/updated-proposal-aave-grants-dao-renewal/11289
Snapshot: https://snapshot.org/#/aave.eth/proposal/0x194b68a5319b074065e38fcb0bd96e8e8b6ce15870844593e79750605a609109

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,7 +21,7 @@ deploy-proposal :; forge script script/DeployMainnetProposal.s.sol:DeployProposa
 
 # Clean & lint
 clean    :; forge clean
-lint     :; npx prettier --write */*.sol
+lint     :; npx prettier --write script/*.sol src/proposals/*/*.sol src/test/proposals/*/*.sol
```
Comment (0xrajath) [inline Makefile:24]: Does `*/*.sol` not work here?

File: remappings.txt [hunk 2]
```diff
@@ -1,4 +1,6 @@
 @forge-std=lib/forge-std/src/
 @openzeppelin=lib/openzeppelin-contracts/contracts/
 @aave-address-book=lib/aave-address-book/src/
-@aave-helpers=lib/aave-helpers/src/
\ No newline at end of file
+@aave-helpers=lib/aave-helpers/src/
+@aave-v2=lib/protocol-v2/contracts/
+@aave-v3=lib/aave-address-book/lib/aave-v3-core/contracts/
```
Comment (0xrajath) [inline remappings.txt:6]: Are you using `aave-v3` libs here?

File: src/proposals/aave-grants-dao-renewal/ProposalPayload.sol [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+contract ProposalPayload {
+    address public constant AGD_MULTISIG = 0x89C51828427F70D77875C6747759fB17Ba10Ceb0;
+    address public constant AAVE = 0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9;
+    address public constant DAI = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    address public constant aDAI = 0x028171bCA77440897B824Ca71D1c56caC55b68A3;
+    address public constant aUSDT = 0x71fc860F7D3A592A4a98740e39dB31d25db65ae8;
+
+    uint256 public constant AMOUNT_DAI = 451_916_320000000000000000;
+    uint256 public constant AMOUNT_ADAI = 1_172_938_790000000000000000;
+    uint256 public constant AMOUNT_AUSDT = 812_944_900000;
```
Comment (0xrajath) [inline src/proposals/aave-grants-dao-renewal/ProposalPayload.sol]: Might be worth it to put comments in terms of USD for additional clarity

**Review by 0xrajath — APPROVED** [review]
LGTM. Left some comments

**Comment by eboadom** [comment]
Some feedback from the BGD side @efecarranza .

**On payload**:
- Better to clarify the new proposed split in terms of assets, as on the forum only USDC and AAVE are mentioned.
- The average should be calculated differently. The norm is 30 days up to the proposal moment. Right now is too short.

**On tests**:
- Redundant files in test/utils folder https://github.com/llama-community/aave-proposals/tree/proposal/src/test/utils
- Redundant files in src/external/aave folder https://github.com/llama-community/aave-proposals/tree/proposal/src/external/aave
- Wrong comment on the amount, should be $812,944.90 (not $812,9422.90) https://github.com/llama-community/aave-proposals/blob/d0bc9cba6678b63708b4c55e275ccfc524a66453/src/proposals/aave-grants-dao-renewal/ProposalPayload.sol#L23

**Comment by eboadom** [comment]
@efecarranza how exactly the average was calculated? Mainly to understand which source and time checkpoints you used

**Comment by efecarranza** [comment]
> @efecarranza how exactly the average was calculated? Mainly to understand which source and time checkpoints you used

I used the Opening Price on Coingecko for the past 30 days.

**Pull request merged**